### PR TITLE
[resolves #2430] Upgrade to wildfly-13.0.0

### LINKED ISSUE #2430: Upgrade to wildfly-13.0.0

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -60,11 +60,16 @@
...
         <version.wildfly.arquillian>2.1.0.Final</version.wildfly.arquillian>
-        <version.wildfly.cxf>3.2.2</version.wildfly.cxf>
-        <version.wildfly.fasterxml.jackson>2.9.4</version.wildfly.fasterxml.jackson>
-        <version.wildfly.infinispan>9.1.6.Final</version.wildfly.infinispan>
+        <version.wildfly.cxf>3.2.4-jbossorg-1</version.wildfly.cxf>
+        <version.wildfly.fasterxml.jackson.core>2.9.5</version.wildfly.fasterxml.jackson.core>
+        <version.wildfly.fasterxml.jackson.datatype>2.9.5</version.wildfly.fasterxml.jackson.datatype>
+        <version.wildfly.fasterxml.jackson.jaxrs>2.9.5</version.wildfly.fasterxml.jackson.jaxrs>
+        <version.wildfly.fasterxml.jackson.module>2.9.5</version.wildfly.fasterxml.jackson.module>
+        <version.com.fasterxml.jackson.dataformat>${version.wildfly.fasterxml.jackson.core}</version.com.fasterxml.jackson.dataformat>
```
Comment (tdiesler) [inline pom.xml]: This version redirect should not be necessary. We like to stick with 'version.wildfly.fasterxml.jackson.core'. In the unlikely case that wildfly diverges versions for datatype, jaxrs, module or dataformat from core, we can introduce additional properties (beside the one for core), which must then map to some property in some wf pom (i.e. the one that defines the divergence.

**Comment by jamesnetherton** [comment]
We should also bump the docker base image version to `openshift/wildfly-130-centos7`.

**Comment by ppalaga** [comment]
> We should also bump the docker base image version to openshift/wildfly-130-centos7.

+1, thanks

**Comment by ppalaga** [comment]
b6cdf50 has `<docker.image.from>openshift/wildfly-130-centos7:latest</docker.image.from>` and returns back to comparing against `wildfly-parent` instead of the new generated `wildfly-component-matrix`. Please re-review.

**Comment by jamesnetherton** [comment]
Can we fix up conflicts. Also, we should bump the project version to 7.0.0-SNAPSHOT.

**Comment by ppalaga** [comment]
ceb1db3 :

* Rebased on top of the latest master
* Added version bump to 7.0.0-SNAPSHOT

**Comment by jamesnetherton** [comment]
ok to test

**Comment by jamesnetherton** [comment]
retest this please

**Comment by jamesnetherton** [comment]
Seems we'll need to fix this https://github.com/wildfly-extras/wildfly-camel/issues/2564.

**Pull request merged**